### PR TITLE
Feat/buy now fe integration

### PR DESCRIPTION
**Pull request by jmulq**
## Description

Buy now fill integration into app. This goes alongside a server PR here https://github.com/Vault-Hill/vhc-marketplace-server/pull/28

### CODE REVIEW

**Review by anjorinjnr — CHANGES_REQUESTED** [review]
Please fix the issue with the BuyNow popup, it's importing some object that doesn't exist. And why do we still have fake data.

I made  a commit that should solve the refresh after checkout,

File: src/__generated/fetchers/AssetFetcher.ts [hunk 1]
```diff
@@ -249,12 +249,5 @@ export const asset$: AssetFetcher<{}, {}> = createFetcher(
   undefined
 )
 
-// prettier-ignore
 export const asset$$ =
```
[the remark this answers is not in the document]
Comment (jmulq) [reply src/__generated/fetchers/AssetFetcher.ts:253]: Amended last commit to resolve this

File: src/fake-data/assets.ts [hunk 2]
```diff
@@ -1,5 +1,3 @@
-import { ListingType } from '../types'
-
 export const assets = [
```
[the remark this answers is not in the document]
Comment (jmulq) [reply src/fake-data/assets.ts:1]: Deleted the fake data we didn't need. Going to let the other guys remove the rest of it as some of the data requires server updates.

**Comment by jmulq** [comment]
Forgot to commit popup changes, they were hidden in my source control app UI. The refresh issue is still occuring, i.e. the buy now flow completes, but the Asset details UI still shows a buy now button.

**Review by anjorinjnr — DISMISSED** [review]

**Pull request merged**